package.json

```diff
@@ -23,46 +23,44 @@
     "fine-uploader-wrappers": "^1.1.0",
     "flatpickr": "4.6.13",
     "lodash": "^4.17.21",
-    "quill": "1.3.7",
+    "quill": "^1.3.7",
```

Comment:
I'm a bit worried by this as there was a reason we locked Quill's version before. I'll take a closer look.

Reply:
Oh there was no release, ok. Quill feels clearly abandoned at this point.

Reply:
Maybe Twill 3 is a good moment to ditch one of the wysiwyg implementations?

Reply:
We will need to keep Quill available as most of our users are relying on it and migrating content would be painful, since, for example, Quill's output includes custom classes for some of its features. If the link feature is completed on Tiptap, I would be willing to set it as the new default.

Reply:
I guess Quill 2 could also be an option once it's done. Anyway, will leave it for now.